File: #2_5월_3주/gaeng2y/FizzBuzz/fizzbuzz.swift
class Solution {
    func fizzBuzz(_ n: Int) -> [String] {
        var res = [String]()
        for i in 1 ... n {
            let str = i % 5 == 0 ? (i % 3 == 0 ? "FizzBuzz" : "Buzz") : (i % 3 == 0 ? "Fizz" : "\(i)")
            res.append(str)
        }
        return res
    }
}

File: #2_5월_3주/agong/FizzBuzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        answer = []
        for n in range(1,n+1):
            if n%3 == 0 and n%5 == 0:
                x = "FizzBuzz"
            elif n%3 == 0:
                x = "Fizz"
            elif n%5 == 0:
                x = "Buzz"
            else:
                x = str(n)
            answer.append(x)
            
        return answer
